lib/cf_obs_binary_builder/templates/libgdiplus.spec.erb

```diff
@@ -0,0 +1,79 @@
+#
+# spec file for SUSE CAP buildpack dependency
+#
+# Copyright (c) 2019 SUSE LINUX GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+
+%define soname 0
+Name:           libgdiplus-<%= version %>
+Version:        <%= version %>
+Release:        1
+Summary:        This is a dependency used in SUSE CAP buildpacks
+License:        (LGPL-2.1+ or MPL-1.1) and MIT
+Group:          Cloud/Platform/scf
+Url:            https://github.com/mono/libgdiplus
+Source:         <%= source %>
+
+# Fixes 64-bit portability issue error
+Patch:          01_pngcodec.patch
+
+BuildRequires:  giflib-devel
+BuildRequires:  libjpeg-devel
+BuildRequires:  libtool
+BuildRequires:  pkg-config
+BuildRequires:  pkgconfig(cairo)
+BuildRequires:  pkgconfig(fontconfig)
+BuildRequires:  pkgconfig(freetype2)
+BuildRequires:  pkgconfig(glib-2.0)
+BuildRequires:  pkgconfig(libexif)
+BuildRequires:  pkgconfig(libpng)
+BuildRequires:  pkgconfig(libtiff-4)
+BuildRequires:  pkgconfig(x11)
+BuildRequires:  pkgconfig(xrender)
+BuildRoot:      %{_tmppath}/libgdiplus-%{version}-build
+
+<%= rpm_macros %>
+
+%description
+Mono library that provide a GDI+ comptible API on non-Windows
+operating systems.
+
+%prep
+%setup -q -n %{name}
+%patch -p1
+sed -i 's,-Wall,-Wall %{optflags},' src/Makefile.am
+
+%build
+autoreconf -fiv
+#%configure
+./configure --prefix=/ \
+    --enable-static
+make %{?_smp_mflags}
+make DESTDIR=%{dependencydir} install %{?_smp_mflags}
+rm -rf %{buildroot}%{_libdir}/%{name}.{a,la}
```

Comment:
Upstream does not seem to do that: https://github.com/cloudfoundry/buildpacks-ci/blob/master/tasks/build-binary-new/builder.rb#L60

Did you compare our dependency result with theirs?

Reply:
It's the same:

```
8497636 drwxr-xr-x 1 ettore users    6 Jan 30 13:56 ..
8497640 -rw-r--r-- 1 ettore users 653K Jan 30 11:47 libgdiplus.a
8497644 -rwxr-xr-x 1 ettore users 1.1K Jan 30 11:47 libgdiplus.la
8497643 lrwxrwxrwx 1 ettore users   19 Jan 30 11:47 libgdiplus.so -> libgdiplus.so.0.0.0
8497639 lrwxrwxrwx 1 ettore users   19 Jan 30 11:47 libgdiplus.so.0 -> libgdiplus.so.0.0.0
8497638 -rwxr-xr-x 1 ettore users 483K Jan 30 11:47 libgdiplus.so.0.0.0
8497641 drwxr-xr-x 1 ettore users   26 Jan 30 11:47 pkgconfig
➜  libgdiplus ls -liah out                                                  
➜  libgdiplus ls -liah libgdi_cf 
total 0
8497890 drwxr-xr-x 1 ettore users   6 Jan 30 14:57 .
8497634 drwxr-xr-x 1 ettore users 192 Jan 30 14:57 ..
8497891 drwxr-xr-x 1 ettore users 162 Jan 17 21:42 lib
➜  libgdiplus ls -liah libgdi_cf/lib 
total 5.0M
8497891 drwxr-xr-x 1 ettore users  162 Jan 17 21:42 .
8497890 drwxr-xr-x 1 ettore users    6 Jan 30 14:57 ..
8497896 -rw-r--r-- 1 ettore users 3.2M Jan 17 21:42 libgdiplus.a
8497895 -rwxr-xr-x 1 ettore users 1.1K Jan 17 21:42 libgdiplus.la
8497894 lrwxrwxrwx 1 ettore users   19 Jan 17 21:42 libgdiplus.so -> libgdiplus.so.0.0.0
8497893 lrwxrwxrwx 1 ettore users   19 Jan 17 21:42 libgdiplus.so.0 -> libgdiplus.so.0.0.0
8497892 -rwxr-xr-x 1 ettore users 1.8M Jan 17 21:42 libgdiplus.so.0.0.0
8497897 drwxr-xr-x 1 ettore users   26 Jan 17 21:42 pkgconfig
```